lib/generators/data_migration/data_migration_generator.rb

```diff
@@ -16,7 +16,8 @@ def create_data_migration
         unless  options.skip_schema_migration?
           migration_template "migration.rb", "db/migrate/#{file_name}.rb"
         end
-        migration_template "data_migration.rb", "db/data/#{file_name}.rb"
+
+        migration_template "data_migration.rb", "db/data/#{file_name}_data.rb"
```

Comment:
Does this `_data` automatically make the class name in the generated file `FooBlahData`?